clarity/contracts/arkadiko-liquidation-rewards-ui-v1-1.clar

```diff
@@ -0,0 +1,100 @@
+;; ---------------------------------------------------------
+;; Maps
+;; ---------------------------------------------------------
+
+(define-map user-tracking
+  { 
+    user: principal
+  } 
+  {
+    last-reward-id: uint
+  }
+)
+
+(define-read-only (get-user-tracking (user principal))
+  (default-to
+    { last-reward-id: u0 }
+    (map-get? user-tracking { user: user })
+  )
+)
+
+;; ---------------------------------------------------------
+;; Claim tracking
+;; ---------------------------------------------------------
+
+(define-public (increase-last-reward-id)
+  (let (
+    (last-id (get last-reward-id (get-user-tracking tx-sender)))
+    (reward-list (map + 
+      (list u0 u1 u2 u3 u4 u5 u6 u7 u8 u9 u10 u11 u12 u13 u14 u15 u16 u17 u18 u19 u20 u21 u22 u23 u24) 
+      (list last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id last-id)
```

Comment:
Q: Might have asked this before, but do we know the costs of this and will it execute?

Reply:
A: I had updated the website about a week ago to use this new contract and it seemed to work fine:
https://explorer.stacks.co/txid/0xdb3c84085fbff205093709397d1680fb204b43af39bfe942cd27148bd5890b2a?chain=mainnet